### PR TITLE
Deepanshutr/splits

### PR DESCRIPTION
**Pull request by deepanshutr**
Adding ownable supply query to splits module.

### CODE REVIEW

File: modules/assets/internal/transactions/mint/keeper.go [hunk 1]
```diff
@@ -56,7 +57,13 @@ func (transactionKeeper transactionKeeper) Transact(context sdkTypes.Context, ms
 		return newTransactionResponse(auxiliaryResponse.GetError())
 	}
 
-	if auxiliaryResponse := transactionKeeper.mintAuxiliary.GetKeeper().Help(context, mint.NewAuxiliaryRequest(message.ToID, assetID, sdkTypes.SmallestDec())); !auxiliaryResponse.IsSuccessful() {
+	split := sdkTypes.SmallestDec()
```
Comment (deepanshutr) [inline modules/assets/internal/transactions/mint/keeper.go:60]: Ignore the next section.

**Review by deepanshutr — COMMENTED** [review]
Self reviewed.

File: modules/splits/internal/key/utilities.go [hunk 2]
```diff
@@ -23,19 +24,29 @@ func readSplitID(splitIDString string) types.ID {
...
+	return splitIDFromInterface(id).OwnableID
 }
 
-func ReadOwnerID(assetID types.ID) types.ID {
-	return splitIDFromInterface(assetID).OwnerID
+func ReadOwnerID(id types.ID) types.ID {
+	return splitIDFromInterface(id).OwnerID
+}
+
+func FromID(id types.ID) helpers.Key {
```
Comment (puneet2019) [inline modules/splits/internal/key/utilities.go:46]: name collides with requests' FromID and ToID.

**Review by puneet2019 — APPROVED** [review]

**Review by avkr003 — APPROVED** [review]

File: modules/splits/internal/key/utilities.go (hunk 2 above)
Comment (deepanshutr) [reply modules/splits/internal/key/utilities.go:46]: True but will be compunded with package name. i.e. key.FromID which will disambiguate

**Pull request merged**